Contributing.md

```diff
@@ -37,46 +37,132 @@
 
 ## Contribution Guidelines
 
-To start contributing, follow steps mentioned below: 
+To start contributing, follow steps mentioned below:
+
+**1.** Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.
+<br />
+
+<img src="imagesContributing/Step1.png" width="100%">
+
+**2.** You'll either find a way to fork to your own account or you will have a link to go to your forked repository. Clicking on either takes you to the forked repository.
+<br />
+
+<img src="imagesContributing/Step2.png" width="100%">
+
+**3.** You can confirm that this repository is a fork of the original repository by finding the "forked from [praveenscience/Internship-LMS-FrontEnd](https://github.com/praveenscience/Internship-LMS-FrontEnd)" text under the repository name.
+<br />
+
+<img src="imagesContributing/Step3.png" width="100%">
+
+**4.** Click on Clone or download button on the forked repository. Make sure you are cloning your version (GitHub.com/\<your username>/Internship-LMS-FrontEnd) and not the CatsInTech organisation's.
+<br />
+
+<img src="imagesContributing/Step4.png" width="100%">
+
+**5.** Clone your forked copy of the project.
 
-**1.**  Fork [this](https://github.com/praveenscience/Internship-LMS-FrontEnd) repository.  
-**2.**  Clone your forked copy of the project.
 ```
 git clone https://<your_user_name>@github.com/<your_user_name>/Internship-LMS-FrontEnd.git
 ```
-**3.** Navigate to the project directory. :file_folder:
+
+<br />
+
+<img src="imagesContributing/Step5.png" width="100%">
```

Comment:
Wrong image? You asked to clone `https://<your_user_name>@github.com/<your_user_name>/` this way but you didn't clone that way in the image.